modules/ROOT/pages/evaluate-dw-expressions.adoc

```diff
@@ -0,0 +1,57 @@
+= DataWeave Expression Watches
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When debugging your Mule application, you may need to evaluate a specific expression while your flow's execution is paused. You can write a DataWeave expression to evaluate or inspect any message property.
```

Comment:
```suggestion
When debugging your Mule application, you must sometimes evaluate a specific expression while your flow's execution is paused. You can write a DataWeave expression to evaluate or inspect any message property.
```
So, you write an expression to evaluate or inspect, and then where does "watch" come in to it?